endpoint-root/endpoint-core/src/main/java/org/divy/common/bo/endpoint/hypermedia/association/Association.java

```diff
@@ -2,45 +2,34 @@
 
 import org.divy.common.bo.BusinessObject;
 import org.divy.common.bo.business.BOManager;
-import org.divy.common.bo.endpoint.hypermedia.association.builder.CreateBuilder;
-import org.divy.common.bo.endpoint.hypermedia.association.builder.UpdateBuilder;
-import org.divy.common.bo.endpoint.hypermedia.association.reader.ReaderBuilder;
+import org.divy.common.bo.mapper.keyvaluemap.KeyValuePairMapper;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.divy.common.bo.mapper.keyvaluemap.KeyValuePairMapper'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)